README.md

```diff
@@ -22,7 +22,6 @@ Jib is a Maven plugin for building Docker and OCI images for your Java applicati
 
 These features are not currently supported but will be added in later releases.
```

Comment:
Would be nice to add additional bullets here.
But, if you don't have any additional features planned, should just combine these into one sentence.